test/reputation-system/root-hash-submissions.js

```diff
@@ -196,7 +198,8 @@ contract("Reputation mining - root hash submissions", (accounts) => {
       );
     });
 
-    it("should allow a user to back the same hash more than once in a same cycle with different entries, and be rewarded", async () => {
+    it.only("should allow a user to back the same hash more than once in a same cycle with different entries, and be rewarded", async () => {
```

Comment:
`it.only` ?? 😱

Reply:
https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-exclusive-tests.md

Hint hint